### PR TITLE
[docs] Add "See Also" blocks for `torch.t` and `torch.transpose`

### LINKED ISSUE #56267: tensor.transpose and tensor.T should have "see also" references to each other.
## 📚 Documentation

https://pytorch.org/docs/stable/generated/torch.transpose.html
and
https://pytorch.org/docs/stable/generated/torch.t.html
should have "see also" references to each other.

I'm willing to make a pull request if the community supports this idea.

cc @brianjo @mruberry

### PR DESCRIPTION
**Pull request by SplitInfinity**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #63506
* __->__ #63461

**Summary**
PR 63177 added "see also" cross-references to `torch.t` and
`torch.transpose`. However, it did so in the form of simple links
preceded by the phrase "See also". A comment on the aforementioned PR
pointed out the RST has special syntax to render a "See Also" box that
is more noticeable. This commit converts the "see also" sentences for
`torch.t` and `torch.transpose` to boxes.

**Test Plan**
Continuous integration.

*Before*

Captura de Pantalla 2021-08-18 a la(s) 10 47 21 a  m

*After*
Captura de Pantalla 2021-08-18 a la(s) 12 32 42 p  m

**Fixes**
This commit fixes #56267.

Differential Revision: [D30799703](https://our.internmc.facebook.com/intern/diff/D30799703)

### CODE REVIEW

**Comment by SplitInfinity** [comment]
I will add screenshots tomorrow morning after the pytorch bot posts a link to the docs rendered using the contents of this commit.

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -9335,6 +9335,10 @@ def merge_dicts(*dicts):
 0-D and 1-D tensors are returned as is. When input is a 2-D tensor this
 is equivalent to ``transpose(input, 0, 1)``.
 
+.. seealso::
+
+        :func:`torch.transpose`.
```
Comment (mruberry) [inline torch/_torch_docs.py:9340]: add a half sentence explaining the difference/similarity

File: torch/_torch_docs.py [hunk 2]
```diff
@@ -9704,6 +9706,10 @@ def merge_dicts(*dicts):
 :attr:`input` tensor, so changing the content of one would change the content
 of the other.
 
+.. seealso::
+
+        :func:`torch.t`.
```
Comment (mruberry) [inline torch/_torch_docs.py:9711]: add a half sentence here, too

File: torch/_torch_docs.py (hunk 1 above)
Comment (SplitInfinity) [reply torch/_torch_docs.py]: Added, will update the screenshots once the preview gets generated.

File: torch/_torch_docs.py (hunk 2 above)
Comment (SplitInfinity) [reply torch/_torch_docs.py]: Added, will update the screenshots once the preview gets generated.

**Review by mruberry — APPROVED** [review]
Thanks @SplitInfinity! Small suggested tweak inline for your review

**Comment by SplitInfinity** [comment]
@splitinfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30799703).

**Pull request closed without merging**